Resources/AzureDevOps/azure-pipelines.yml

```diff
@@ -4,8 +4,32 @@ variables:
   BuildPlatform: 'any cpu'
   BuildConfiguration: 'release'
 
+resources:
+  repositories:
+  - repository: self
+  - repository: dfc-devops
+    type: github
+    name: SkillsFundingAgency/dfc-devops
+    endpoint: 'GitHub (ESFA)'

```

Comment:
Worth adding a 
    ref: refs/tags/v1.2.0
property here to fix the version that's used